pyproject.toml

```diff
@@ -66,7 +66,7 @@ pyyaml = "*"           # Base requirement
 scipy = ">1.2"         # Base requirement, requires openblas
 xarray = "<2024.3"     # Base requirement; Pinned to <2024.3 due to bug mentioned in #558
 astropy = "*"          # Used for accounting leap seconds when converting epoch timestamps
-cartopy = ">=0.22.0"   # No longer requires libgeos
+cartopy = ">=0.23.0"   # No longer requires libgeos
```

Comment:
I think at least make a note that 0.22 works, but0.23 required for test outputs to pass.

I still don't feel right forcing the version JUST for test outputs to pass.